lightwood/data/statistical_analysis.py

```diff
@@ -12,6 +13,27 @@
 from lightwood.data.cleaner import _clean_float_or_none
 
 
+def get_datetime_histogram(data, data_dtype):
+    if isinstance(data[0], float) or isinstance(data[0], int):
+        data = [_clean_float_or_none(x) for x in data]
+    else:
+        data = [_clean_float_or_none(parse_dt(str(x)).timestamp()) for x in data]
+
+    Y, X = np.histogram(data, bins=min(50, len(set(data))),
+                        range=(min(data), max(data)), density=False)
+    if data_dtype == dtype.integer:
+        Y, X = np.histogram(data, bins=[int(round(x)) for x in X], density=False)
+
+    X = X[:-1].tolist()
+    Y = Y.tolist()
+
+    X = [datetime.datetime.fromtimestamp(x) for x in X]
```

Comment:
Do we have a unit-test to demonstrate this works as intended?

Reply:
Hmh, no, but I can add histogram checking to some unittests